x/genutil/client/cli/validate_genesis_test.go

```diff
@@ -6,9 +6,16 @@ import (
 
 	"github.com/stretchr/testify/require"
 
+	appmodulev2 "cosmossdk.io/core/appmodule/v2"
+	"cosmossdk.io/x/staking"
```

Comment:
The conflicts are fixed and this will be merged. But @ziscky will remove the staking dependency in a follow-up and we'll backport (ref https://github.com/cosmos/cosmos-sdk/pull/21701#issuecomment-2348528111)